packages/docs/docs/agent/push.md

```diff
@@ -14,19 +14,82 @@ While the Medplum Agent traditionally connected healthcare facilities to cloud s
    - Relay a DICOM SR to a RIS using DICOM.
 4. **Simplified Infrastructure**: Eliminate the need for site-to-site VPNs, making setups more straightforward and secure.
 
+## How it works
+
+First, consider the most simple case: an HL7 device on the network:
+
+![Medplum Agent Push 1](./medplum-agent-push-1.webp)
+
+We can emulate this setup using the Medplum CLI:
+
+```bash
```

Comment:
I think it would read more cleanly if you put the emulation code for each step _before_ the diagram